src/shell/commands/data_operations.cpp

```diff
@@ -1540,6 +1540,7 @@ bool copy_data(command_executor *e, shell_context *sc, arguments args)
                                            {"no_overwrite", no_argument, 0, 'n'},
                                            {"no_value", no_argument, 0, 'i'},
                                            {"geo_data", no_argument, 0, 'g'},
+                                           {"no_ttl", no_argument, 0, 'e'},
```

Comment:
you declare `no_ttl` here,  but options  declare `return_expire_ts`, I suggest they should be coincident

Reply:
+1

Reply:
Actually I think `no_ttl` and `return_expire_ts` are different. `no_ttl` tells that by default ttl will be kept while we use shell to copy data. `return_expire_ts` is used not only for copy data,  in fact  at any time that `get_scanner_request` is called the developer should decide whether `expire_ts` is needed. For now, at most cases,  `expire_ts` is not needed by the caller of `get_scanner_request`, therefore I set `return_expire_ts` as ture to tell server that `expire_ts` should be returned. However, for the most cases the callers just ignore `return_expire_ts` and the server only return key/value without `expire_ts` which is not needed.

Reply:
Okay, I have been persuaded, you can wait for @Shuo-Jia 's opinion.

Reply:
Yeah, I see